### PR TITLE
Add fundraising 2021 banner to home page (Fixes #10700)

### LINKED ISSUE #10700: Fundraising banner for Mozilla Foundation
## Description

[Issue for previous banner for reference](https://github.com/mozilla/bedrock/issues/6693#)

Functionality can stay stay the same. We will update the design and background image.

Mofo has requested a fundraising banner for two dates:
1. Nov 30 (giving tuesday)
2. Dec 21 - 31

---

### Design
Mockup images in google drive: [desktop & mobile](https://drive.google.com/drive/folders/1VmFhMURm0ipFwc8k2nRTuHybcibgmSAx?usp=sharing)
Banner image in google drive: [blue banner](https://drive.google.com/open?id=1VcBPsENRH-VcJT8IFPlPhzJ2xZpRb9xg&authuser=sabrina%40mozillafoundation.org&usp=drive_fs)
Figma files: [mockups](https://www.figma.com/file/2TK8tXeDIiLC67TMdbMIcO/?node-id=115%3A270) & [prototypes](https://www.figma.com/proto/2TK8tXeDIiLC67TMdbMIcO/Fundraising?page-id=115%3A270&node-id=116%3A625&viewport=501%2C48%2C0.25&scaling=scale-down&starting-point-node-id=116%3A625&show-proto-sidebar=1)

### Copy
Used Giving Tuesday copy from the No Testing section for mockups: [google doc](https://docs.google.com/document/d/1r2Ft9tvEZLzgBPQE3HzECTUhCMBlNiSvWyFJCJOKsTQ/edit?usp=sharing)
This contains text for both Giving Tuesday (November 30), and Year-End of December (between 12/20/21 and 12/31/21) banners.

#### Success Criteria
- [ ] Launch Giving Tuesday version on Nov 30
- [ ] Turn off Giving Tuesday Version on Wed 1
- [ ] Turn on Holiday version on Dec 21
- [ ] Turn off Holiday version on Dec 31
- [ ] Switch

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds 2021 fundraising banner behind the switch `SWITCH_FUNDRAISING_BANNER_EOY2021`.
- Adds secondary "Giving Tuesday" headline behind additional switch `SWITCH_FUNDRAISING_BANNER_GIVING_TUESDAY_2021`.

## Issue / Bugzilla link
#10700

## Testing
When set in `.env`:
- `SWITCH_FUNDRAISING_BANNER_EOY2021=on`
- `SWITCH_FUNDRAISING_BANNER_GIVING_TUESDAY_2021=on`:

Verify:
- [ ] http://localhost:8000/en-US/ should show banner with new copy / new visuals (headline mentions "Giving Tuesday").
- [ ] http://localhost:8000/de/ should show banner with old copy / new visuals.
- [ ] http://localhost:8000/fr/ should show banner with old copy/ new visuals.
- [ ] http://localhost:8000/es-ES/ should show banner with old copy / new visuals.
- [ ] Locales that do not appear in the [donate list](https://github.com/mozilla/bedrock/blob/master/bedrock/settings/base.py#L880) should all default to $USD.

When set in `.env`:
- `SWITCH_FUNDRAISING_BANNER_GIVING_TUESDAY_2021=on`
- `SWITCH_FUNDRAISING_BANNER_GIVING_TUESDAY_2021=off`

Verify:
- [ ] http://localhost:8000/en-US/ should show banner with new copy / new visuals (headline mentions "holiday season").

When set in `.env`:
-`SWITCH_FUNDRAISING_BANNER_EOY2021=off`

Verify:
- [ ] All home pages should not show the fundraising banner.

### CODE REVIEW

**Comment by alexgibson** [comment]
This is ready for code review, but still **do not merge** pending stakeholder approval.

**Comment by craigcook** [comment]
Some locales seem to be pulling the numeric amounts from the params but not the symbol or format so they're defaulting to dollars.

Example, Russian should be  "X ₽" but is showing "$X" :
image

I'm not sure what the cause is, but Russian looks correct in the old banner on dev:
image

Checking a few other locales, most don't have the banner at all but hi-IN does, and has the same issue both on local with the new banner and dev with the old banner so now I don't even know what to think.
image
image

**Comment by alexgibson** [comment]
@craigcook I filed an issue for most of these. `/ru/` has been fixed, so you may need to pull the latest L10n files to see it fixed locally. I'll add `hi-IN` to the issue for Peiying, thanks!

**Comment by alexgibson** [comment]
Issue is here if that helps: https://github.com/mozilla-l10n/www-l10n/issues/254

File: l10n/en/banners/fundraising.ftl
Comment (craigcook) [inline l10n/en/banners/fundraising.ftl:14]: Should be a curly apostrophe:
```suggestion
banner-fundraising-body-v2 = We’re proudly nonprofit, working to keep the web healthy. Your contributions help build a safe and open internet.
```

**Review by craigcook — APPROVED** [review]
only the teeniest of nitpicks, but I know Peiying would flag it. R+wc! 💰

File: bedrock/base/templates/includes/banners/fundraiser.html
Comment (craigcook) [inline bedrock/base/templates/includes/banners/fundraiser.html:34]: [Late-breaking change request](https://github.com/mozilla/bedrock/issues/10700#issuecomment-972258794) to set the default back to One-time

**Review by craigcook — CHANGES_REQUESTED** [review]

**Comment by alexgibson** [comment]
Thanks @craigcook updated. Looks like Peiying has fixed the currency L10n bugs as well so I think we're all set here. Going to merge as soon as Circle is happy 👍

**Comment by alexgibson** [comment]
@craigcook ah, looks like I still need you to approve the last change so I'll leave merging to you. Thanks

**Review by craigcook — APPROVED** [review]

**Pull request merged**
